packages/extensions-sdk/src/cli/commands/create.ts

```diff
@@ -177,7 +176,7 @@ async function createLocalExtension({
 
 function getPackageManifest(name: string, options: ExtensionOptions, deps: Record<string, string>) {
 	const packageManifest: Record<string, any> = {
-		name: EXTENSION_NAME_REGEX.test(name) ? name : `directus-extension-${name}`,
```

Comment:
Same as in `packages/extensions-sdk/src/cli/commands/add.ts`:

> I think it would make sense to revert this change, following 35714f6a4efe67d071df3de9d6bc8a814bded5dd